clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/CTInAppNativeInterstitialFragment.java

```diff
@@ -63,15 +50,23 @@ public class CTInAppNativeInterstitialFragment extends CTInAppBaseFullNativeFrag
 
     private GifImageView gifImageView;
 
-    private ExoPlayer player;
-
-    private StyledPlayerView playerView;
+    private InAppVideoPlayerHandle handle;
 
     private RelativeLayout relativeLayout;
 
     private FrameLayout videoFrameLayout;
 
-    private ViewGroup.LayoutParams videoFramelayoutParams, playerViewLayoutParams, imageViewLayoutParams;
+    private ViewGroup.LayoutParams imageViewLayoutParams;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (VideoLibChecker.INSTANCE.mediaLibType == VideoLibraryIntegrated.MEDIA3) {
+            handle = new Media3Handle();
```

Comment:
Can be renamed to videoHandle/ mediaHandle

Reply:
keeping it specific to lib used for now.